docs/content/cookbook/oss-deployment-guide.md

```diff
@@ -0,0 +1,625 @@
+        uses: appleboy/ssh-action@master
+        env:
+          GITHUB_SHA: ${{ github.sha }}
+        with:
+          host: ${{env.IP_ADDRESS}}
+          username: "root"
+          key: ${{ secrets.SSH_KEY }}
+          port: "22"
+          envs: GITHUB_SHA
+          script: |
+            mkdir -p "/var/www/html/releases/${GITHUB_SHA}"
+            tar xzf /var/www/html/artifacts/${GITHUB_SHA}.tar.gz -C "/var/www/html/releases/${GITHUB_SHA}"
+            rm -rf /var/www/html/artifacts/${GITHUB_SHA}.tar.gz
+```
+::list{type="success"}
+ - We download the tarball that we created in the previous step.
+ - We then upload the tarball to our server.
+ - We then extract the tarball on our server.
+ - We then remove the tarball from our server.
+::
+:hr
+Now that we've prepared our server, we can activate the release. We'll do this by creating a symlink to the new release and restarting our services.
+
+### Activate the release
+```yaml
+activate-release:
+    name: "Activate release"
+    runs-on: ubuntu-latest
+    needs: prepare-release-on-servers
+    steps:
+      - name: Activate Release
+        uses: appleboy/ssh-action@master
+        env:
+          RELEASE_PATH: /var/www/html/releases/${{ github.sha }}
+          ACTIVE_RELEASE_PATH: /var/www/html/live
+        with:
+          host: ${{env.IP_ADDRESS}}
+          username: "root"
+          key: ${{ secrets.SSH_KEY }}
+          port: "22"
+          envs: RELEASE_PATH,ACTIVE_RELEASE_PATH
+          script: |
+            ln -s -n -f $RELEASE_PATH $ACTIVE_RELEASE_PATH
+            systemctl restart vsfcommunity-server
+            systemctl restart vsfcommunity-web
+            chown -R www-data:www-data ${RELEASE_PATH}
+```
+::list{type="success"}
+ - We create a symlink to the new release.
+ - We restart our services.
+ - We change the owner of the release directory to www-data. This is the user that our services run as.
+::
+:hr
+If we were to deploy our application now, it would work. However, we would quickly run out of disk space on our server. This is because we're not cleaning up old releases. Let's add a step to clean up old releases.
+
+### Clean up old releases
+```yaml
+clean-up:
+    name: "Clean up old versions"
+    runs-on: ubuntu-latest
+    needs: activate-release
+    steps:
+      - name: clean up old releases
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{env.IP_ADDRESS}}
+          username: "root"
+          key: ${{ secrets.SSH_KEY }}
+          port: "22"
+          script: |
+            cd /var/www/html/releases && ls -t -1 | tail -n +4 | xargs rm -rf
+            cd /var/www/html/artifacts && rm -rf *
+      - uses: geekyeggo/delete-artifact@v1
+        with:
+          name: app-artifacts
+``` 
+:hr
+### Full Script
+In the root of your project directory, create a file called `.github/workflows/deploy.sh` and paste the following contents into it:
+
+```yaml[.github/workflows/deploy.sh]
```

Comment:
Side question, would it be helpful if the docs had support for like toggleable content? So large code blocks or other secondary content could be hidden behind a toggle?

Reply:
Maybe. I only worry that if things are toggleable some text might be overlooked if it's behind a toggle. I guess I'd have to see it to know.